insights_dart/lib/src/user_token_storage.dart

```diff
@@ -1,5 +1,7 @@
 import 'package:hive/hive.dart';
+import 'package:path_provider/path_provider.dart';
 import 'package:uuid/uuid.dart';
+import 'dart:io' show Platform;
```

Comment:
This should work for native platforms, however, `dart:io` isn't available for web :/